HeliPort/Appearance/it.lproj/Localizable.strings

```diff
@@ -0,0 +1,65 @@
+"About HeliPort" = "Su HeliPort";
+"Address: " = "Indirizzo：";
```

Comment:
Unchanged Chinese Style colon `：`, please replace it with a normal colon with a space `: `